backend/migrations/2023-06-20-110200_plants_relations/up.sql

```diff
@@ -0,0 +1,12 @@
+-- Your SQL goes here
+CREATE TYPE RELATIONS_TYPE AS ENUM ('companion', 'neutral', 'antagonist');
+
+CREATE TABLE relations (
+    plant1 TEXT NOT NULL,
+    plant2 TEXT NOT NULL,
```

Comment:
Unique names not needed in this table, ids should be enough?

```suggestion
```

Reply:
I will set this as resolved for now. We can discuss it via the other comment below.

Reply:
Which comment discussed this?

Is there a specific reason why you would want to store the unique name?

Reply:
If it's just about the CSV import, there is no issue for me to write the insert script to look up the ID first. But it seems like we already decided on that further down?